e2e/supp-benefits/src/mspsb-supp-benefits.po.ts

```diff
@@ -0,0 +1,125 @@
+import { browser, by, element, WebElement, protractor, Key } from 'protractor';
+import { PersonalInfoPageTest, ContactInfoPageTest } from './mspsb-supp-benefits.data';
+import { BaseMSPTestPage } from '../../msp.po';
+/**
+ * This class is for GENERAL functions, and all those that target components
+ * from the moh-common-lib.  The long-term plan will be to move these over to
+ * `moh-common-lib/testing` once created. That way different Angular projects
+ * can use the same e2e starting board.
+ */
+export class BaseMSPSuppBenefitsTestPage extends BaseMSPTestPage {
```

Comment:
You can remove BaseMSPSuppBenefitsTestPage as well as all the other "BaseMSP*TestPage" classes, you only need to keep BaseMSPTestPage

For example in this case, there's really nothing that BaseMSPSuppBenefitsTestPage adds.  The PreparePage could directly inherit from BaseMSPTestPage.